### DISCUSSION TITLE
NIST 800-171 rules? CMMC update?

### DISCUSSION THREAD

**Discussion by DavidKorpiewski** [wazuh/wazuh#16881] [General]
Hello my friends!
I am using Wazuh to attempt to achieve NIST 800-171 (and eventually CMMC) compliance.  I found some feature requests for CMMC, but not NIST 800-171.  However, I know we could extrapolate most of the NIST 800-171 from 800-53.  
I have a few questions:

(1) Is there a dashboard plugin to show compliance for NIST 800-171 for Wazuh?
(2) If there is, how would I install it?
(3) Any updates on potential CMMC compliance plugin for the dashboard?

Thank you very much and be well!
David

**Comment by mdiego92** [comment]
Hello @DavidKorpiewski,

Currently, it is not possible to create a custom compliance group. I would recommend opening an issue requesting this feature with our team in the following repository: https://github.com/wazuh/wazuh

Even if Wazuh does not yet evaluate NIST SP 800-171 certification or any equivalent, we have the SOC2 certification.
There are some areas of overlap between SOC 2 and NIST SP 800-171, particularly around security controls.

Regarding your CMMC inquiry, the related information about it is located in this issue: https://github.com/wazuh/wazuh/issues/7658 and there is no ETA for the moment.

Please let me know if you have any remaining questions.
Regards!
